protoc-gen-swagger/main.go

```diff
@@ -0,0 +1,115 @@
+package main
```

Comment:
Originally I thought about adding a flag to protoc-gen-grpc-gateway to let it generate swagger schema.
What do you think about that?  Do you think it is better to make it an independent protoc plugin?